subjects/reversestrings/READMED.md

```diff
@@ -0,0 +1,38 @@
+## reverse-strings
+
+### Instructions
+
+Write a function that takes a slice of strings and returns a single string containing them in reverse order with a space between each element of the slice,if the slice is empty, return an empty string.
```

Comment:
Write a function that takes a slice of strings and returns a single string containing them in reverse order with a space between each element of the slice. If the slice is empty, return an empty string.

Reply:
fixed